README.md

```diff
@@ -1,96 +1,154 @@
-In practice, it will act as a staging ground for some APIs that may eventually be incorporated into the Swift Standard Library, and when that happens such changes will be proposed to the Swift Standard Library using the established evolution process of the Swift project.
 
-Swift Numerics uses GitHub issues to track bugs and features. We use pull requests for development.
+Swift Numerics is a standalone library that is separate from the core Swift
+project. In practice, it will act as a staging ground for some APIs that may
+eventually be incorporated into the Swift Standard Library. When that happens,
+such changes will be proposed to the Swift Standard Library using the
+established evolution process of the Swift project.
+
+Swift Numerics uses GitHub issues to track bugs and features. We use pull
+requests for development.
+
+### How to propose a new module
 
-To propose a new module:
 1. Raise an issue with the [new module] tag.
 2. Raise a PR with an implementation sketch.
-3. Once you have some consensus, ask an admin to create a feature branch against which PRs can be raised.
-4. When the design has stabilized and is functional enough to be useful, raise a PR to merge the new module to master.
+3. Once you have some consensus, ask an admin to create a feature branch against
+   which PRs can be raised.
+4. When the design has stabilized and is functional enough to be useful, raise a
+   PR to merge the new module to master.
+
+### How to propose a new feature for an existing module
 
-To propose a new feature for an existing module:
 1. Raise an issue with the [enhancement] tag.
 2. Raise a PR with your implementation, and discuss the implementation there.
-3. Once there is a consensus that the new feature is desirable and the design is suitable, it can be merged.
+3. Once there is a consensus that the new feature is desirable and the design is
+   suitable, it can be merged.
 
-To fix a bug, or make smaller improvements:
-1. Raise a PR with your change. Be sure to add test coverage for whatever changes you are making.
+### How to fix a bug, or make smaller improvements
 
-Questions about how to use Swift Numerics modules, or issues that are not clearly bugs can be discussed in the ["Swift Numerics" section of the Swift forums.](https://forums.swift.org/c/related-projects/swift-numerics)
+1. Raise a PR with your change. 
+2. Make sure to add test coverage for whatever changes you are making.
+
+### Forums
+
+Questions about how to use Swift Numerics modules, or issues that are not
+clearly bugs can be discussed in the ["Swift Numerics" section of the Swift
+forums](https://forums.swift.org/c/related-projects/swift-numerics).
 
 ## Modules
-1. [RealModule](Sources/RealModule/README.md)
-2. [ComplexModule](Sources/ComplexModule/README.md)
+
+1. About [`RealModule`](Sources/RealModule/README.md)
+    - [`ApproximateEquality`](Sources/RealModule/ApproximateEquality.swift)
```

Comment:
We should definitely link to detail on ApproximateEquality, but we should link to a markdown document, rather than the implementation. Let's remove this for now and open an issue to track adding one.

Reply:
Thanks! I removed the link and opened an issue: https://github.com/apple/swift-numerics/issues/153